eos/form-factors/parametric-bgl1997-impl.hh

```diff
@@ -37,7 +37,9 @@ namespace eos
         _chi_1m( 5.131e-04), // TODO remove hard-coded numerical values
```

Comment:
These values should ideally be moved to `mesonic-processes.hh`

Reply:
Yes, but that's a post-merge cleanup, since it's already in the master branch.

Reply:
Understood